src/arrayMethodSort.js

```diff
@@ -3,9 +3,34 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = undefined) {
+    for (let i = 0; i < this.length - 1; i++) {
+      if (String(this[i]) > String(this[i + 1])) {
+        const currentIndex = this[i];
+
+        this[i] = this[i + 1];
+        this[i + 1] = currentIndex;
+
+        i = -1;
+      }
+    }
```

Comment:
and remove this loop after that

Reply:
The default function was too long, so I wrote it below.